cli/docker_build.py

```diff
@@ -184,7 +184,7 @@ def null_swaparoo():
 
 class Docker(object):
 
-    def docker_build(self, dockerUtilsObj, appObj, directory, repo, projects, path, image_tag, docker_file='Dockerfile'):
+    def docker_build(self, dockerUtilsObj, appObj, directory, repo, projects, path, image_tag, build_args, docker_file='Dockerfile'):
```

Comment:
If there are no build-args (meaining no key-value pairs), then it will just proceed with the usual build that has happened so far, thus I initialed it to an empty dictionary.